pyproject.toml

```diff
@@ -12,7 +12,7 @@ documentation = "https://docs.beamerbridge.com/"
 [tool.poetry.dependencies]
 python = ">=3.10,<3.11"
 web3 = "^6.5.0"
-click = "^8.1.4"
+click = "=8.1.3"
```

Comment:
do we really need = and why we are doing this?

Reply:
> do we really need = and why we are doing this?

I would say yes, for the time being because the click update to 8.1.4 broke a bunch of things:
https://app.circleci.com/pipelines/github/beamer-bridge/beamer/3466/workflows/db56e0fb-aa9c-48b3-8179-0e3f3d78f7a6/jobs/33514

I would prefer to address the conversion to new click later.